idc_index/index.py

```diff
@@ -341,7 +422,7 @@ def get_series_file_URLs(self, seriesInstanceUID):
         return file_names
 
     def get_viewer_URL(
-        self, seriesInstanceUID, studyInstanceUID=None, viewer_selector=None
+        self, seriesInstanceUID=None, studyInstanceUID=None, viewer_selector=None
```

Comment:
This is potentially confusing, since it implies both series and study UIDs can be None.

Reply:
I do not know there if there is any other way. 

I notice that it is implemented the same way in `download_from_selection` function as well, where we do not explicitly require any UIDs.
https://github.com/ImagingDataCommons/idc-index/blob/main/idc_index/index.py#L505
![image](https://github.com/ImagingDataCommons/idc-index/assets/115020590/6eb33ef3-69d9-4674-beae-3541b112fcaa)

Reply:
Yes, good point. I agree there is no good solution.